### ISSUE TITLE
Usecases UI Improvement

### ISSUE THREAD

**Issue by Monilprajapati** [Shuffle/Shuffle#1274]
**Is your feature request related to a problem? Please describe.**
Currently, the UI of the Usecases section on the Usecase page is just the basic one. Whenever we click on any particular usecase, it goes bigger and shows more details about that usecase, which does not look more user-friendly.

**Describe the solution you'd like**
I will redesign and redevelop that section and make it like Doc-type UI. Which means all the use cases will be on the left side, and whenever we click on any use case, the details of that use case will open on the right side of that section. For example, you can think about the docs section, but here that type of section will be created for all 5 usecases and their subcases.

@frikky  @0x0elliot I would love to work on this issue and will make it smooth and easy to use to get a better user experience. Assign it to me, and I will do it.

**Screenshots of where and how**

See, All these subcases (e.g., Email management, EDR to ticket, etc..) will go on the left side in sidebar and their content will be visible on the part like the docs section.

Current UI : 
![image](https://github.com/Shuffle/Shuffle/assets/99136041/1b848804-3820-49ff-8fd0-2dd4e0ebb216)

![image](https://github.com/Shuffle/Shuffle/assets/99136041/06632475-a333-450b-8d03-a4bf26026b53)

**Comment by 0x0elliot** [comment]
Hey @Monilprajapati what do you want the UI to look like then? Do you have a design in your head that you would want to show us?

**Comment by Monilprajapati** [comment]
> Hey @Monilprajapati what do you want the UI to look like then? Do you have a design in your head that you would want to show us?

I haven't created it yet, but it's completely clear in my mind. See you know the document section in shuffle. So each usecase will have its own section, in which there will be one sidebar on the left side of that usecase section for their subcases, like email management, etc., and the right section will show it's details, like on the docs page of the Shuffle website.

**Comment by Monilprajapati** [comment]
Like this :
![image](https://github.com/Shuffle/Shuffle/assets/99136041/bd7f4cfa-86b4-4cfc-b7c6-bee69b8bebbf)

Every usecase will have this type of section, and that sidebar will consist of their subcases, and the right side will contain the details of it.
Let me explain you by taking the example for one usecase
So this is use case 1:
![image](https://github.com/Shuffle/Shuffle/assets/99136041/f3e93e74-5e74-447f-bed4-0e78824b52c7)

Now in the new design, on the left side, it will contain a list of the subcases, like email management and all:
Image 1:
![image](https://github.com/Shuffle/Shuffle/assets/99136041/280cd576-79df-4eb4-afdd-26fb64d6c470)

and by clicking on any subcases the content of that subcase will open on the right side of the section : 
Image : 2
![image](https://github.com/Shuffle/Shuffle/assets/99136041/9ebb227c-76cd-49a1-bb73-dd247b9bb70c)

each usecases will have image 1 and 2 for their subcases and their content

**Comment by frikky** [comment]
Hey!

This is not a bad idea at all, but both the UI & UX of the sidebar + main body would have to be made very easy to interact with. As the main content itself is the usecases, moving it to the side may move the focus away a bit too much, but this again just comes down to how it's implemented

@iiAnna what do you think? :)

**Comment by iiAnna** [comment]
I agree, it is not a bad idea. Maybe a prototype can help with understanding and testing the flow.
